Microsoft.Toolkit.Forms.UI.XamlHost/Microsoft.Toolkit.Forms.UI.XamlHost.csproj

```diff
@@ -50,6 +63,16 @@
     </Reference>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
+    <PackageReference Include="System.Runtime.WindowsRuntime" Version="4.6.0-preview1-27018-01" />
```

Comment:
We will need to review the right packages once we have the official contracts from the SDK published in NuGet

Reply:
Do we have an ETA on that?